fgd/bases/TriggerOnce.fgd

```diff
@@ -3,9 +3,9 @@
 	line(255 255 255, targetname, filtername) 
 = TriggerOnce
 	[
-	spawnflags(flags)  =
+	spawnflags(flags) =
 		[
-		1: "Clients/Players" : 1
+		1: "Clients/Players" : 0
```

Comment:
Intentional? It was 1 all through the commit history.

Reply:
It is yes. I thought the idea was to only have it on by default for teleports so i figured this was incorrect. On second look though it is on for other source games so ill revert that.

Reply:
Fixed this and also moved the ghosts spawnflag next to clients/players as they're very similar.